shuffle/shuffle_test.go

```diff
@@ -64,3 +70,82 @@ func shuffleTest(suite Suite, k, N int) {
 		}
 	}
 }
+
+func sequenceShuffleTest(suite Suite, k, NQ, N int) {
```

Comment:
Pass the `*testing.B` in this method. Then you can use the `b.N`, and `require.NoError(b, err)`.

Reply:
It can't. This function is used for the unit and benchmark tests, which is also why we use `panic` and not `require.NotError`. Nothing new though, it follows the same pattern as the existing tests.